src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -340,7 +340,7 @@
         size (fs/size file)
         sha256 (ks/file->sha256 (io/file file))
         ;; we trust the file path from Puppet, so extract the subpath from file
-        static-path (re-find (re-pattern (str #"[^/]+/[a-z][a-z0-9_]*/(?:tasks|files|lib)/.*" basename)) file)
+        static-path (re-find (re-pattern (str #"[^/]+/[a-z][a-z0-9_]*/(?:tasks|files|scripts|lib)/.*" basename)) file)
```

Comment:
Somewhat unrelated to this PR, but it'd be nice to move the static pattern (and its coercion to a string) into a def.